web/src/engine/websites/MangaRomance.ts

```diff
@@ -0,0 +1,64 @@
+import { Tags } from '../Tags';
+import icon from './MangaRomance.webp';
+import { Chapter, DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchJSON, FetchRequest } from '../FetchProvider';
+
+type FeedResults = {
+    feed: {
+        entry: {
+            link: {
+                rel: string,
+                href: string,
+                title: string
+            }[],
+        }[]
+    }
+}
+
+function MangaInfoExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname.split('/').pop(),
+        title: anchor.text.trim()
+    };
+}
+
+@Common.MangasSinglePageCSS('', 'div.sidebar div#Label1 div.list-label-widget-content ul li a', MangaInfoExtractor)
+@Common.PagesSinglePageCSS('article div.entry-content img')
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('mangaromance', `Manga Romance`, 'https://www.mangaromance.eu', Tags.Media.Manga, Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Language.English, Tags.Source.Aggregator);
```

Comment:
Cannot reach the website, is this region locked?

Reply:
Just tested, its slow but works.